scanner/scanner.go

```diff
@@ -344,54 +266,32 @@ func (s *Scanner) Scan(ctx context.Context, foundCert func(*ct.LogEntry), foundP
 		close(stop)
 	}()
 
-	fetches := make(chan fetchRange)
+	batches := make(chan EntryBatch) // Output from the Fetcher.
 	jobs := make(chan entryInfo, s.opts.BufferSize)
```

Comment:
While we're here can we use something less generic than `jobs`?

Reply:
Renamed to `entries`.